### PR TITLE
fix: save url operation is delayed

### PR DESCRIPTION
**Pull request by sywhb**
normal save url operation could be delayed by jobs from lower priority queue so we deprioritize these jobs by setting priority to 10

### CODE REVIEW

File: packages/api/src/utils/createTask.ts [hunk 1]
```diff
@@ -37,6 +42,37 @@ import View = google.cloud.tasks.v2.Task.View
...
+/**
+ * we want to prioritized jobs by the expected time to complete
+ * lower number means higher priority
+ * priority 1: jobs that are expected to run immediately
+ * priority 5: jobs that are expected to run in less than 10 seconds
+ * priority 10: jobs that are expected to run in less than 1 minute
+ * priority 50: jobs that are expected to run in less than 30 minutes
+ * priority 100: jobs that are expected to run in less than 1 hour
+ **/
+export const getJobPriority = (jobName: string): number => {
```
Comment (jacksonh) [inline packages/api/src/utils/createTask.ts:54]: Ah this is nice. One thing I'd say is we should probably log when we hit default so we handle jobs not having priority set properly.

**Comment by sywhb** [comment]
> One question on this that I couldn't tell: currently we have two ways a feed can be refreshed: either from the cron job or by a user adding it. When the user adds it we want to give a high priority refresh. Do we lose that with this change?

i fixed the bug in the previous code. the user-added rss feed was used to be given a lower priority(higher number)

File: packages/api/src/jobs/rss/refreshAllFeeds.ts [hunk 2]
```diff
@@ -129,15 +132,15 @@ type QueuePriority = 'low' | 'high'
 export const queueRSSRefreshFeedJob = async (
   jobid: string,
   payload: any,
-  options = { priority: 'high' as QueuePriority }
```
Comment (sywhb) [inline packages/api/src/jobs/rss/refreshAllFeeds.ts:132]: @jacksonh the default priority previously was `high` which was assigned to user added rss feed refresh job

File: packages/api/src/jobs/rss/refreshAllFeeds.ts [hunk 3]
```diff
@@ -129,15 +132,15 @@ type QueuePriority = 'low' | 'high'
...
+  options = { priority: 'low' as QueuePriority }
 ): Promise<Job | undefined> => {
   const queue = await getBackendQueue()
   if (!queue) {
     return undefined
   }
-  return queue.add('refresh-feed', payload, {
+  return queue.add(REFRESH_FEED_JOB_NAME, payload, {
     jobId: `${jobid}_${JOB_VERSION}`,
-    priority: options.priority == 'low' ? 10 : 50,
```
Comment (sywhb) [inline packages/api/src/jobs/rss/refreshAllFeeds.ts:140]: @jacksonh the priority for `high` is larger than `low` and this means user added rss feeds were processed after cronjob triggered jobs

**Pull request merged**